bitex/pairs.py

```diff
@@ -211,6 +211,15 @@ def bter_formatter(base, quote):
         """Format currencies for BTer."""
         return base.lower() + '_' + quote.lower()
 
+    @staticmethod
+    def poloniex_unformatter(pair):
```

Comment:
This isn't valid. While tethers are coupled with US Dollars, they are not USD. Poloniex only trades tethered USD, and this should be clear to the user.